src/features/unified-share-modal/SharedLinkSection.js

```diff
@@ -130,7 +130,13 @@ class SharedLinkSection extends React.Component<Props, State> {
             this.setState({ isAutoCreatingSharedLink: false });
         }
 
-        if (Browser.canWriteToClipboard() && triggerCopyOnLoad && !isAutoCreatingSharedLink && sharedLink.url) {
+        if (
+            Browser.canWriteToClipboard() &&
+            triggerCopyOnLoad &&
+            !isAutoCreatingSharedLink &&
+            sharedLink.url &&
+            isCopySuccessful === null
```

Comment:
this check was missing previously and the tests for rejections werent going into the catch block due to test structure